src/test/java/org/broadinstitute/hellbender/tools/exome/alleliccount/AllelicCountUnitTest.java

```diff
@@ -17,7 +17,7 @@
     @Test
     public void testEstimateMinorAlleleFraction() {
         final SimpleInterval interval = new SimpleInterval("contig", 1, 2);
-        final double tolerance = 0.0001;
+        final double tolerance = 0.01;
```

Comment:
I think we might want to choose a slightly smaller tolerance (0.001-0.005 is fine, I think).  0.0001 was probably overkill though...